exchange/utils.go

```diff
@@ -442,11 +442,18 @@ func buildRequestExtForBidder(bidder string, requestExt json.RawMessage, request
 }
 
 func buildRequestExtAlternateBidderCodes(bidder string, accABC *openrtb_ext.ExtAlternateBidderCodes, reqABC *openrtb_ext.ExtAlternateBidderCodes) *openrtb_ext.ExtAlternateBidderCodes {
+	if len(bidder) == 0 {
+		return nil
+	}
+
+	normalisedBidderName, _ := openrtb_ext.NormalizeBidderName(bidder)
```

Comment:
Need to keep in mind this returns an empty string for request aliases. Is that ok here?

Reply:
This line completely changed in [1244d1a](https://github.com/prebid/prebid-server/pull/3229/commits/1244d1a96ae7c0276dc25b576a7e10e0e3afabdb)